### DISCUSSION TITLE
What does "experimental" mean in "USB host controller (UHC) driver API"?

### DISCUSSION THREAD

**Discussion by Himajin009** [zephyrproject-rtos/zephyr#69954] [Q&A]
Hello.
According to the document of "USB host controller (UHC) driver API", UHC driver API is experimental.
What does "experimental" mean in this document?
For example, does it mean some of device class are still not supported or some other functions are not supported?

And please let me know about a roadmap or plan that zephyr project supports UHC officially.

Thank you.

**Comment by GeorgeCGV** [comment]
Hi, the progress can be observed at https://github.com/zephyrproject-rtos/zephyr/issues/42066/.

**Reply by Himajin009** [reply]
Thank you for information.
I will close this discussion.
